tests/test_interfaces_usb.py

```diff
@@ -16,6 +22,16 @@ def test_interface_usb_io(strategy, shell):
     # Write tmp file onto usb device
     shell.run_check(f"dd if=/tmp/test_file of=/dev/{usb_device} bs=1M count=15")
 
+    # Disconnect and connect the USB stick to make sure all buffers have been flushed.
+    strategy.eet.link("")
+    time.sleep(1)
+    strategy.eet.link("USB1_IN -> USB1_OUT")
+    time.sleep(5)
```

Comment:
Out of curiosity: what happens when you drop the sleeps?

Reply:
Shouldn't we disconnect the USB device after the test?

Reply:
Without the first sleep Linux does not recognize that the USB-device was gone at all. I guess the disconnect is too short for the USB-stick to reset.

Without the second sleep the block device is not reliably ready when trying to `dd` from there.

So this has the potential to be racy :smile:

Reply:
> Shouldn't we disconnect the USB device after the test?

Totally right! I've moved the *eet*-reset into a fixture. so that it will always be reset.